week-1/I-floats/exercise.js

```diff
@@ -1,2 +1,6 @@
 var numberOfStudents = 15;
 var numberOfMentors = 8;
+var stprecent=(numberOfStudents*100)/23;
+var mentpercent=(numberOfMentors*100)/23;
+console.log("Percentage students: "+Math.round(stprecent));
+console.log("Percentage mentors: "+Math.round(mentpercent));
```

Comment:
Here's a good example of being consistent with an existing variable naming scheme. If the variable was named`percentOfStudents` I would have a good idea of what it represented without having to read the whole line of code.